pyproject.toml

```diff
@@ -1,7 +1,85 @@
 [build-system]
 requires = [
-    "cython>=0.29.29",
+    "Cython~=0.29.29",
     "oldest-supported-numpy",
-    "setuptools",
-    "wheel",
+    "setuptools>=61",
 ]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "Fiona"
+dynamic = ["readme", "version"]
+authors = [
+    {name = "Sean Gillies"},
+]
+maintainers = [
+    {name = "Fiona contributors"},
+]
+description = "Fiona reads and writes spatial data files"
+keywords = ["gis", "vector", "feature", "data"]
+license = {text = "BSD 3-Clause"}
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Scientific/Engineering :: GIS",
+]
+
+requires-python = ">=3.7"
+dependencies = [
+    "attrs>=19.2.0",
+    "certifi",
+    "click~=8.0",
+    "click-plugins>=1.0",
+    "cligj>=0.5",
+    "munch>=2.3.2",
+]
+
+[project.optional-dependencies]
+all = ["Fiona[calc,s3,test]"]
+calc = ["shapely"]
+s3 = ["boto3>=1.3.1"]
+test = [
+    "Cython~=0.29.29",
```

Comment:
Is there a reason that `test` needs cython?

Reply:
No, this is a leftover from old versions, I think.

Reply:
I removed it in 10ce6b7